docs/GettingStarted/QuickstartTutorial.md

```diff
@@ -284,33 +370,51 @@ Now, we should also have data in the `Mean Time to Restore` metric in Grafana da
 
 **Change Failure Rate** is a key quality metric that measures what percentage of changes fails in the production environment. It is crucial to have alignment on what constitutes a failure. The recommended definition is a change that either results in degraded service or subsequently requires remediation.
 
-<!-- ## Partially Automated Demo
```

Comment:
https://github.com/konveyor/pelorus/issues/768 can we delete and close this issue or its better to add this information in some place?

Reply:
The e2e demo is an automated helper, but really don't want to cover it now as this would require more testing and possibly adjusting e2e demo script to match operator workflow...

Reply:
ok